README.md

```diff
@@ -62,16 +98,15 @@ Then, you'll need to add the `infinity-loader` component to your template, like
 {{infinity-loader infinityModel=model}}
 ```
 
-Now, whenever the `infinity-loader` component is in view, it will send an action to the route.  This method uses action bubbling, which may not be the preferred way of passing data around your application.  See [Closure Actions](#ClosureActions).
 
-When the new records are loaded, they will automatically be pushed into the Model array.
+In both cases, whenever the `infinity-loader` component is in view, we will fetch the next page for you.
 
 Lastly, by default, `ember-infinity` expects the server response to contain something about how many total pages it can expect to fetch. `ember-infinity` defaults to looking for something like `meta: { total_pages: 20 }` in your response.  See [Advanced Usage](#AdvancedUsage).
```

Comment:
Can we give this note it's own heading? We get a lot of issues from folks who forgot to add meta to their response, and this applies to both approaches listed above